tutorial_sarscov2.md

```diff
@@ -0,0 +1,259 @@
+
+> **For advanced users:** If your are fluent with these tools, you can:
+> - directly download and install
+>   [bioconda](https://bioconda.github.io/user/install.html) and
+>   [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html#installation-via-conda),
+> - clone the [sars-cov2 branch of V-pipe](https://github.com/cbg-ethz/V-pipe/tree/sars-cov2)
+> - and start using V-pipe with them, using the `--use-conda` to
+>   [automatically download and install](https://snakemake.readthedocs.io/en/stable/snakefiles/deployment.html#integrated-package-management)
+>   any [further pipeline dependencies]({{ "/pipeline/" | relative_url }}).
+>
+> The present tutorial will show simplified commands that automate much of this process.
+
+
+To deploy V-pipe, you can use the installation script with the following parameters:
+
+```bash
+curl -O 'https://raw.githubusercontent.com/cbg-ethz/V-pipe/master/deploy_vpipe.sh'
+bash deploy_vpipe.sh -b sars-cov2 -p testing -w work
+cd ./testing/work/
+```
+
+ - using `-p` specifies the subdirectory where to download and install snakemake and V-pipe
+ - using `-b` specifies which branch of V-pipe to use
+
+   > **Versions**: different versions come with different defaults.
+   > - `sars-cov2` branch is adapted for SARS-CoV-2
+   > - the default branch (`master`) is adapted for HIV
+   >
+   > As the project matures, [release tar-balls frozen at specific versions](https://github.com/cbg-ethz/V-pipe/releases) will progressively be made available, see the [Cluster example below](#cluster-deployment).
+   >
+ - using `-w` will create a working directory and populate it. It will copy over the references and the default `vpipe.config`, and create a handy `vpipe` short-cut script to invoke `snakemake`.
+
+   > **Tips**:
+   > To create and populate other new working directories, you can call `init_project.sh` from within the new directory :
+   ```bash
+   mkdir -p working_2
+   cd working_2
+   ../V-pipe/init_project.sh
+   ```
+
+## Running V-pipe
+
+
+Copy the `samples` directory you created in the step [Preparing a small dataset](#preparing-a-small-dataset) to this `working` directory. 
+You can display the directory structure with `tree samples` or `find samples`[^fancyfind].
+
+[^fancyfind]: `find samples | sed -e "s/[^-][^\/]*\// |/g" -e "s/|\([^ ]\)/|-\1/"` is overly long but produces slightly prettier output than `find samples`
+
+Check the parameters of the `vpipe.config` file with you favorite editor (`vim`, `emacs`, `nano`, [butterflies](https://xkcd.com/378/), etc.).
+For SNVs and Local (windowed) haplotype reconstruction, you will need at least the following options:
+
+*[SNVs]: Single Nucleotide Variants
+
+```
+[input]
+reference = references/NC_045512.2.fasta
+
+[output]
+snv = True
+local = True
+global = False
+
+[general]
+aligner = bwa
+```
+
+Check what will be executed:
+```bash
+./vpipe --dryrun -p --cores 2
```

Comment:
This is not needed here `-p --cores 2`

Reply:
Right. Too much copy-pasting while to much blood detectable in the coffein-vascular system...

Reply:
... also comes from my habit of just adding "--dry-run" in most other tools (such as `rsync`) before a complex command to see its impact.